R/create_template.R

```diff
@@ -127,11 +129,11 @@ create_template <- function(
   }
 
   if (is.null(office) | office == "") {
-    subdir <- paste0("~/stock_assessment_reports/report")
+    subdir <- paste0(file_dir,"/stock_assessment_reports/report")
```

Comment:
These commands with `paste0(..."/")` will fail depending on your operating system. Please use `file.path()` or [`fs::path()`](https://fs.r-lib.org/reference/path.html), where I prefer the latter but that is just personal preference. These functions allow you to just specify the words in the path rather than the separator, where the separator will be automatically generated and operating system specific. This comment carries out through the entire code base.

Reply:
Thanks for that advice Kelli! I did not know that and appreciate the help